### DISCUSSION TITLE
How Long Will 3.7.X Be Supported?

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#15970] [Q&A] [answered]
I see that there are releases for 3.7.X every 2-3 weeks, and see the 4.0.0 release coming soon. But am not sure if there's a specific cut off date that 3.7 will not be supported any further, with all work moving to 4.0.0?

**Comment by candlerb** [comment, marked as answer]
I think that's today. Over about 7 years, I've never seen Netbox release a X.Y.Z after X.(Y+1).0 or (X+1).0.0 has been released.

**Comment by jeffgdotorg** [comment]
Great question, @mr1716. Up to and including today, @candlerb is correct. We currently don't plan any more 3.7.x releases, though we might deviate from that norm under extraordinary circumstances.
